### DISCUSSION TITLE
Soviet 2-seat Lunar Module

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#761] [Q&A]
Uh oh, I may have made a mistake on the Soviet two-seat lunar lander.  We changed the name from Duet to LKM in version 1.0.  Something caught my attention this morning, and I can't find a reference to "LKM" anywhere online - and I don't recall where that came from.  I _think_ I got it from [http://www.astronautix.com/r/russiasovielunarprojects.html](url), but I don't see it there anywhere now.  And I'm wondering if I got it twisted up in my head.

![LunarLanderProposalsNumbered](https://user-images.githubusercontent.com/3118014/227614576-163e0ebf-2602-4816-9228-360d64a57934.png)
These are the lander proposals from that page (the bottom row on the page shows lunar _base_ components).  
**1** is the original L3 proposal, the inspiration for our C-Kicker and Soyuz Lunar Landing.  
**2** is the LK, the lander they actually built.  
**3** is the LK-3, a halfhearted direct ascent idea, 
**4** is the LK-700, the Soviets' more serious direct ascent vehicle.  
**5** and **6** are L3M project proposals for two-man landers, and 
**7** is the LEK, proposed to be launched on a new rocket, the (first proposed) Vulkan.

As it happens, the two on the right look most like the 2-seater in the game, and the left-hand one of those (**6**) would have also been a two-man lander as part of N1M-L3M, an expanded N1-L3 program.  I think I might have confused L3M into LKM in my head.  Maybe because the LK was the actual lander part of the N1-L3 program, so I extrapolated that the lander for the L3M project would be designated LKM?

I hate to leave a historically inaccurate name lying around, so changing LKM to L3M sounds tempting - though it would be a little messy since LKM is already in version 1.1 and in our documentation.  On the other hand, maybe LKM _is_ the best name for it, reasoning that the lander for the L3M _project_ would be called LKM...in which case maybe we should leave it as is.   In which case the game would stay the same; I'd just need to update the documentation a little.

**Comment by hweimer** [comment]
[This page](http://www.astronautix.com/d/dlbbeaconlander.html) also refers to a LKM lunar lander, so keeping the name should be fine. I think your argument about differentiating between the entire spacecraft (L3M) and the lunar lander component (LKM) sounds reasonable.

**Reply by peyre** [reply]
Thanks.  The question above went through several iterations.  I started out saying hey, maybe I made a mistake and need to change the name, but as I looked it over more closely and thought about it I came around to thinking LKM would be a good extrapolation from the historic proposal.  Thanks, I'll update the explanation for the name change in my RIS guide.  (I had said "the documentation" above, thinking the manual also mentioned it, so nothing here on github needs updating.  Handy that.)

**Comment by Angel-0a** [comment]
I just found this webpage referencing LKM lander in detail: https://graphicsnickstevens.substack.com/ Looks like you didn't imagine it. There is a catch though...

There are two interesting articles on LKM. [1973 N1-L3M Project Details](https://graphicsnickstevens.substack.com/p/1973-n1-l3m-project-details) is a translation of the N1-L3M Project schedule. At the bottom of the page there is an image of the original project flow chart, where you can clearly see LKM section. The second one is [Nikolaev, LKM Moon lander, and GIRD](https://graphicsnickstevens.substack.com/p/nikolaev-lkm-moon-lander-and-gird), which talks about LKM lander in more detail. And it seems that LKM landers were intended to be direct return landers, assembled from two parts in Earth orbit, akin to the Soyuz Lunar Landing mission depicted in the game. This similarity is further underscored by the fact that the LKM landers incorporated a Soyuz descent module!

**Reply by peyre** [reply]
Thanks Angel!  It's nice to get confirmation of this.
